src/Controllers/ContentController.php

```diff
@@ -0,0 +1,638 @@
+use Railroad\Railcontent\Services\CommentService;
+use Railroad\Railcontent\Services\ConfigService;
+use Railroad\Railcontent\Services\ContentService;
+use Railroad\Railcontent\Services\FullTextSearchService;
+use Railroad\Railcontent\Support\Collection;
+
+class ContentController extends Controller
+{
+    /**
+     * @var ContentService
+     */
+    private $contentService;
+
+    /**
+     * @var CommentService
+     */
+    private $commentService;
+
+    /**
+     * @var VimeoVideoSourcesDecorator
+     */
+    private $vimeoVideoDecorator;
+
+    /**
+     * @var UserProviderInterface
+     */
+    private $userProvider;
+
+    /**
+     * @var MailService
+     */
+    private $mailoraMailService;
+
+    /**
+     * @var ContentHierarchyRepository
+     */
+    private $contentHierarchyRepository;
+
+    /**
+     * @var FullTextSearchService
+     */
+    private $fullTextSearchService;
+
+    /**
+     * @var StripTagDecorator
+     */
+    private $stripTagDecorator;
+
+    /**
+     * @var DownloadService
+     */
+    private $downloadService;
+
+    /**
+     * ContentController constructor.
+     *
+     * @param ContentService $contentService
+     * @param CommentService $commentService
+     * @param VimeoVideoSourcesDecorator $vimeoVideoDecorator
+     * @param UserProviderInterface $userProvider
+     * @param MailService $mailoraMailService
+     * @param ContentHierarchyRepository $contentHierarchyRepository
+     * @param FullTextSearchService $fullTextSearchService
+     * @param StripTagDecorator $stripTagDecorator
+     * @param DownloadService $downloadService
+     */
+    public function __construct(
+        ContentService $contentService,
+        CommentService $commentService,
+        VimeoVideoSourcesDecorator $vimeoVideoDecorator,
+        UserProviderInterface $userProvider,
+        MailService $mailoraMailService,
+        ContentHierarchyRepository $contentHierarchyRepository,
+        FullTextSearchService $fullTextSearchService,
+        StripTagDecorator $stripTagDecorator,
+        DownloadService $downloadService
+    ) {
+        $this->contentService = $contentService;
+        $this->commentService = $commentService;
+        $this->vimeoVideoDecorator = $vimeoVideoDecorator;
+        $this->userProvider = $userProvider;
+        $this->mailoraMailService = $mailoraMailService;
+        $this->contentHierarchyRepository = $contentHierarchyRepository;
+        $this->fullTextSearchService = $fullTextSearchService;
+        $this->stripTagDecorator = $stripTagDecorator;
+        $this->downloadService = $downloadService;
+    }
+
+    /**
+     * @param $contentId
+     * @param Request $request
+     * @return JsonResponse
+     * @throws NonUniqueResultException
+     */
+    public function getContent($contentId, Request $request)
+    {
+        $content = $this->contentService->getById($contentId);
+
+        if (!$content) {
+            return response()->json();
```

Comment:
I think a proper custom exception may be thrown, instead of having second empty response return statement.